pkg/certs/types.go

```diff
@@ -0,0 +1,48 @@
+package certs
+
+import (
+	"strings"
+
+	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+)
+
+type ConfigMapRefByNamespaceName []certgraphapi.InClusterConfigMapLocation
```

Comment:
followup, I'm in favor of putting these in a library-go util package.

Reply:
> ollowup, I'm in favor of putting these in a library-go util package.

err, not these, the sorting functions

Reply:
Right, I'll update https://github.com/openshift/library-go/pull/1592 and remove those on library-go bump